homeassistant/components/stream/fmp4utils.py

```diff
@@ -151,7 +151,7 @@ def find_moov(mp4_io: BufferedIOBase) -> int:
     while 1:
         mp4_io.seek(index)
         box_header = mp4_io.read(8)
-        if len(box_header) != 8:
+        if len(box_header) != 8 or box_header[0:4] == b"\x00\x00\x00\x00":
```

Comment:
On a general note, this looks like something that should be in a separate library, not in Home Assistant Core 🤷‍♂️

Reply:
We've discussed moving this out of core, and I think we generally agreed it makes sense, but haven't made the plan in more detail or executed on it yet.